src/components/accession2/view/Accession2View.tsx

```diff
@@ -343,11 +373,11 @@ export default function Accession2View(props: Accession2ViewProps): JSX.Element
             </Box>
           </Box>
         )}
-        {accession?.storageLocation && (
+        {accession?.facilityId && (
           <Box sx={editableParentProps}>
             <Icon name='iconMyLocation' className={classes.iconStyle} />
             <Box sx={editableProps}>
-              <Typography paddingLeft={1}>{accession.storageLocation}</Typography>
+              <Typography paddingLeft={1}>{getSeedBank(organization, accession.facilityId)?.name}</Typography>
```

Comment:
We are showing the seed bank only now, and not the storage location. Is that what we want?

Reply:
i can change this to `seed bank name (italic storage location name)` and we can iterate  on that?

Reply:
Added storage location, we can change the display later if Ming has another suggestion.

<img width="414" alt="Terraware App 2022-09-20 10-27-29" src="https://user-images.githubusercontent.com/1865174/191325185-ec14ecf1-8461-4a84-ada5-36c464790ed6.png">

Reply:
Ming commented that we need to show both.

Reply:
Thanks!

Reply:
I asked if my current proposal "seed bank name (storage location name)" is ok. if she suggests something else I will send a new PR